TF_Settings_Web/src/Components/Pages/Camera/QuickSetup/Calibration/CalibrationComponents.tsx

```diff
@@ -79,18 +81,48 @@ export const CalibrationProgressCircle: React.FC<CalibrationProgressCircleProps>
     </div>
 );
 
-export const CalibrationHandLostMessage = () => (
-    <div className={'hand-not-found-container'}>
-        <img src={HandIcon} alt="Hand Icon" />
-        <p>Hand Not Detected</p>
-    </div>
-);
+interface HandsLostProps {
+    timeToPosSelect?: number;
+}
+
+const ReturnToPositionScreenMessage: FC<HandsLostProps> = ({ timeToPosSelect }) => {
+    if (!timeToPosSelect) {
+        return <></>;
+    }
+
+    const timeToPosSelectLength = timeToPosSelect.toString().length;
+    const timeoutLength = TIMEOUT_S.toString().length;
+
+    const numString = ''.padEnd(timeoutLength - timeToPosSelectLength, '0') + timeToPosSelect;
+
+    const formattedString = [...numString].map((char, index) => {
```

Comment:
Might be worth a comment here of an example string and why we are doing it to save us brainpower in the future

Reply:
Good shout